### ISSUE TITLE
Review of Wheat module.

### ISSUE THREAD

**Issue by ApsimBot** [APSIMInitiative/APSIMClassic#701]
Legacy Bug ID: 1183
Author: Chris 
Date: 2010-08-10 10:16:06 +0000 UTC

The Steering Commitee has requested that the Reference Panel (RP) use the new process for incorporating modifications/improvements into APSIM to undertake a major review the 3 key crop modules in APSIM, those being Wheat, Sorghum and Chickpea.

	The RP has nomimated Karine Chenu and Neil Huth to develop a case (i.e. documentation, validation data etc.) for Wheat.

	Once this case has been developed and submitted, a Reviewer/s will be appointed.

**Comment by ApsimBot** [comment]
Author: karine.chenu@deedi.qld.gov.au
Date: 2010-08-10 12:20:00 +0000 UTC

Subject: Review of Wheat module - (DO NOT EDIT THIS:1183)
	Dear Karine and Neil,
	

	 

	The APSIM Initiative Steering Committee has requested that a major review be undertaken of the 3 key crop modules in APSIM, those being Wheat, Sorghum and Chickpea. 

	 

	 

	 

	

	Given your respective fields of scientific expertise, the Reference Panel (RP) believes that you would be the most appropriate to work together to develop a detailed case (i.e. a formal proposal incorporating issues such as development history, science documentation, validation, priorities for future improvements etc.) for Wheat.

	 

	

	If you can confirm that you are able to undertake this task and what would be a realistic timeframe it would be greatly appreciated.

	 

	Once this formal proposal has been developed and submitted, a reviewer/s will be appointed by the RP.  For your information, the link for this new task is http://www.apsim.info/BugTracker/edit_bug.aspx?id=1183 .

	 

	I have also attached a document which is intended as a guide for developers and reviewers which may be of assistance in this process. 

	 

	I look forward to hearing from you.

	 

	Regards

	Chris Murphy 
	Principal Project Officer (APSIM)
	Agri-science Queensland
	Department of Employment, Economic Development & Innovation 
	203 Tor Street, Toowoomba, Queensland 4350, Australia
	PO Box 102, Toowoomba, Queensland 4350, Australia
	t: +61 7 4688 1596 
	f: +61 7 4688 1193 
	e: apsim@dpi.qld.gov.au 
	w: www.dpi.qld.gov.au 
	
	The APSIM (Agricultural Production systems siMulator) Initiative is an unincorporated joint 
	venture between the Commonwealth Scientific and Industrial Research Organisation, 
	the State of Queensland and The University of Queensland. 
	Website www.apsim.info

attachment: Improvements to APSIM - Instructions to Reviewers and Developers.pdf       savesize: 129436   content-type: application/pdf

**Comment by ApsimBot** [comment]
Author: murphy
Date: 2011-05-04 09:33:00 +0000 UTC

[APSIM Winter Cereals Working Group 18.02.11 Actions.doc](https://www.apsim.info/BugTracker/view_attachment.aspx?download=0&id=32625&bug_id=1183)
Size: 55296

**Comment by ApsimBot** [comment]
Author: murphy
Date: 2011-05-05 11:29:00 +0000 UTC

From: Doherty, Alastair 

	Sent: Tuesday, 3 May 2011 5:00 PM

	To: Murphy, Chris

	Subject: Update on Progress of wheat model documentation for AJV Refence Panel

	Hi Chris,

	An update on where we are for this task for the Refence panel

	Task: To complete a document describing the scienctific processes that are currently implemented in the APSim wheat module

	Group members: Al Doherty, Bangyou Zheng, Karine Chenu, Niel Huth

	Bangyou and I have been looking into the code and preparing documents describing the process implented in the APSIm wheat module. We met with a larger group several weeks ago and decided that Bangyou's document was going to be an excellent platform to submit these findings and he is also happy to share his work. I have met with Bangyou since and have made plans to incorporate the missing pieces and have made progress. I am currently reviewing the document for any omissions.

	Once this is done Karine and Niel will also review the document and Niel will also add some documentation regarding the validation of the wheat module.

	We are hopeful that this document should be finalised for release to the Reference Panel at next months RP meeting.

	Cheers

	Al

	 

	Al Doherty
	Senior Scientist (Programmer)
	Agricultural Systems Modelling
	Agri-Science Queensland (DEEDI)
	Agricultural Production Systems Research Unit (APSRU)
	Telephone: +61 (0)7 4688 1566 Mobile: +61 (0)434364522 Fax: +61 (0)7 4688 1193
	Email: alastair.doherty@deedi.qld.gov.au
	Website: www.dpi.qld.gov.au  Call Centre: 13 25 23

**Comment by ApsimBot** [comment]
Author: ver078
Date: 2012-01-17 16:16:00 +0000 UTC

Reviewer: 
	Working Group
	
	Review Panel Contact: 
	Graeme Hammer

**Comment by ApsimBot** [comment]
Author: murphych
Date: 2015-03-13 09:26:00 +0000 UTC

Updated wheat module documentation now available on the APSIM web site
